docs/buildah-manifest-add.md

```diff
@@ -82,4 +102,4 @@ buildah manifest add --arch arm64 --variant v8 mylist:v1.11 docker://fedora@sha2
 ```
 
 ## SEE ALSO
-buildah(1), buildah-manifest(1), buildah-manifest-create(1), buildah-manifest-remove(1), buildah-manifest-annotate(1), buildah-manifest-inspect(1), buildah-manifest-push(1), buildah-rmi(1)
+buildah(1), buildah-login(1), buildah-manifest(1), buildah-manifest-create(1), buildah-manifest-remove(1), buildah-manifest-annotate(1), buildah-manifest-inspect(1), buildah-manifest-push(1), buildah-rmi(1),docker-login(1)
```

Comment:
nit of a nit, can you add a space before docker-login(1) please?
```suggestion
buildah(1), buildah-login(1), buildah-manifest(1), buildah-manifest-create(1), buildah-manifest-remove(1), buildah-manifest-annotate(1), buildah-manifest-inspect(1), buildah-manifest-push(1), buildah-rmi(1), docker-login(1)
```

Reply:
Good catch, I fixed that